src/Behat/Symfony2Extension/ServiceContainer/Symfony2Extension.php

```diff
@@ -125,7 +126,7 @@ public function process(ContainerBuilder $container)
                 require_once($bootstrap);
             } elseif (file_exists($bootstrapPath)) {
                 require_once($bootstrapPath);
-            } else {
+            } elseif ($bootstrapPath != self::DEFAULT_KERNEL_BOOTSTRAP) {
```

Comment:
please use strict comparison

Reply:
Sure, can do. What's the difference to you?